src/MainWindow.vala

```diff
@@ -136,4 +141,26 @@ public class Installer.MainWindow : Hdy.Window {
             accounts_service.keyboard_layouts = layouts;
         }
     }
+
+    private async void set_left_handed () {
+        AccountsService accounts_service = null;
+
+        try {
+            var act_service = yield GLib.Bus.get_proxy<FDO.Accounts> (GLib.BusType.SYSTEM,
+                                                                      "org.freedesktop.Accounts",
+                                                                      "/org/freedesktop/Accounts");
+            var user_path = act_service.find_user_by_name (account_view.created.user_name);
+
+            accounts_service = yield GLib.Bus.get_proxy (GLib.BusType.SYSTEM,
+                                                        "org.freedesktop.Accounts",
+                                                        user_path,
+                                                        GLib.DBusProxyFlags.GET_INVALIDATED_PROPERTIES);
+        } catch (Error e) {
+            warning ("Unable to get AccountsService proxy, clock format on new user may be incorrect: %s", e.message);
```

Comment:
Do we need to repeat ourselves with all this? It seems like it would make sense to put the left-handed setting in the same method with the keyboard and locale settings above